apps/public-docsite-v9/src/DocsComponents/FluentDocsPage.stories.tsx

```diff
@@ -46,7 +46,8 @@ const useStyles = makeStyles({
     display: 'flex',
   },
   description: {
-    display: 'flex',
+    display: 'grid',
```

Comment:
Grid is used because I can't style directly `<Description />` component from SB

Reply:
wondering how all of this will work with storybook 7 , cc @dmytrokirpa